lldb/source/Commands/CommandObjectStart.cpp

```diff
@@ -0,0 +1,102 @@
+//===-- CommandObjectStart.cpp -------------------------------------------===//
```

Comment:
Revert this source file since we are just adding an option to "process launch" right?